src/main/java/org/openstreetmap/atlas/utilities/vectortiles/MinimumZoom.java

```diff
@@ -0,0 +1,166 @@
+    private static final int TEN = 10;
+    private static final int TWELVE = 12;
+    private static final int THIRTEEN = 13;
+    private static final int FOURTEEN = 14;
+
+    private MinimumZoom()
+    {
+        // Util
+    }
+
+    static int amenity(final String tagValue)
+    {
+        switch (tagValue)
+        {
+            case "hospital":
+                return TWELVE;
+            default:
+                return THIRTEEN;
+        }
+    }
+
+    static int building(final String tagValue)
+    {
+        return THIRTEEN;
+    }
+
+    static int highway(final String tagValue)
+    {
+        switch (tagValue)
+        {
+            case "motorway":
+            case "trunk":
+                return SIX;
+            case "primary":
+                return EIGHT;
+            case "secondary":
+                return NINE;
+            case "tertiary":
+                return TEN;
+            case "motorway_link":
+            case "trunk_link":
+            case "primary_link":
+            case "secondary_link":
+            case "residential":
+            case "unclassified":
+                return TWELVE;
+            case "service":
+            case "living_street":
+            case "pedestrian":
+            case "bridleway":
+            case "footway":
+            case "cycleway":
+            case "track":
+            case "steps":
+                return THIRTEEN;
+            default:
+                return FOURTEEN;
+        }
+    }
+
+    static int railway(final String tagValue)
+    {
+        switch (tagValue)
+        {
+            case "rail":
+            case "light_rail":
+                return EIGHT;
+            case "tram":
+            case "subway":
+                return TWELVE;
+            case "disused":
+                return FOURTEEN;
+            default:
+                return THIRTEEN;
+        }
+    }
+
+    static int route(final String tagValue)
+    {
+        switch (tagValue)
+        {
+            case "ferry":
+                return EIGHT;
+            default:
+                return FOURTEEN;
+        }
+    }
+
+    static int place(final String tagValue)
+    {
+        switch (tagValue)
+        {
+            case "country":
+                return THREE;
+            case "state":
+                return FIVE;
+            default:
+                return TWELVE;
+        }
+    }
+
+    static int natural(final String tagValue)
+    {
+        switch (tagValue)
+        {
+            case "glacier":
+                return EIGHT;
+            default:
+                return TWELVE;
+        }
+    }
+
+    static int leisure(final String tagValue)
+    {
+        switch (tagValue)
+        {
+            case "park":
+                return EIGHT;
+            default:
+                return THIRTEEN;
+        }
+    }
+
+    static int landuse(final String tagValue)
+    {
+        switch (tagValue)
+        {
+            case "basin":
```

Comment:
If we're going to keep this in the code, might make sense to leverage the specific tag class enums here instead of hard-coding the values.

Reply:
Unfortunately, you need to have Strings as constants at compile time for them to work in switches...

Reply:
For example, say I do this

```java
private static final String HOSPITAL = AmenityTag.HOSPITAL.name().toLowerCase();
    private static final String MOTORWAY = HighwayTag.MOTORWAY.getTagValue();
    private static final String TRUNK = HighwayTag.TRUNK.getTagValue();
    private static final String PRIMARY = HighwayTag.PRIMARY.getTagValue();
```
and then have

```java
    static int amenity(final String tagValue)
    {
        switch (tagValue)
        {
            case HOSPITAL:
                return TWELVE;
            default:
                return THIRTEEN;
        }
    }
```

Well, that doesn't work... Won't compile...

Reply:
Static block?

Also why go to lower case?

Reply:
Because the actual tag values are lower case, and the enum itself is caps. If you look at .getTagValue, it's actually doing that, plus putting it in intern. The only thing with that is, only some of the tag enums have that implemented...